store-api-server/src/payment/controller/payment.controller.ts

```diff
@@ -125,7 +125,7 @@ export class PaymentController {
     @Body('currency') currency: string = BASE_CURRENCY,
     @Body('recreateNftOrder') recreateNftOrder: boolean = false,
   ): Promise<PaymentIntent> {
-    return this.paymentService.handleCreatePaymentIntent({
+    return await this.paymentService.handleCreatePaymentIntent({
```

Comment:
Is the `await` needed? (this isn't a blocker for merging so feel free to ignore, I'm just interested why it's needed)

Reply:
I am just afraid when it is missing. I have seen a weird corner case with throwing errs and catching where it will behave unexpected without the `await`. None of that happens here but I am just afraid whenever there is an await missing.